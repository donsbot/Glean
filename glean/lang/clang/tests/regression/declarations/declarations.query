query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
