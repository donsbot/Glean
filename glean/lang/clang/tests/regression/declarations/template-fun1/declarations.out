[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 22,
      "lineEnd": 11,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 26,
      "lineEnd": 9,
      "columnEnd": 28
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 13,
      "columnBegin": 22,
      "lineEnd": 15,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 13,
      "columnBegin": 27,
      "lineEnd": 13,
      "columnEnd": 30
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 17,
      "columnBegin": 1,
      "lineEnd": 21,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 17,
      "columnBegin": 8,
      "lineEnd": 17,
      "columnEnd": 12
    }
  }
]